server/sonar-server/src/main/java/org/sonar/server/user/index/UserIndex.java

```diff
@@ -108,7 +112,7 @@ public UserDoc getNullableByLogin(String login) {
       .setFetchSource(
         new String[] {UserIndexDefinition.FIELD_LOGIN, UserIndexDefinition.FIELD_NAME},
         null)
-      .setQuery(QueryBuilders.filteredQuery(QueryBuilders.matchAllQuery(), filter));
+      .setQuery(QueryBuilders.filteredQuery(matchAllQuery(), filter));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "filteredQuery"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)